src/swift/SwiftAPIView.xcodeproj/project.pbxproj

```diff
@@ -421,6 +421,7 @@
 				DEVELOPMENT_TEAM = ZQB5E94758;
 				ENABLE_HARDENED_RUNTIME = YES;
 				INFOPLIST_FILE = "$(SRCROOT)/Info.plist";
+				MACOSX_DEPLOYMENT_TARGET = 10.15;
```

Comment:
Note the value is set on project level, not target level.

Reply:
Fixed.